core/primitives-core/src/runtime/fees.rs

```diff
@@ -240,8 +249,17 @@ pub fn transfer_send_fee(
         (_, AccountType::NamedAccount) => transfer_fee,
         // No account will be created, just a regular transfer.
         (false, _) => transfer_fee,
-        // Currently, no account is created on transfer to ETH-implicit account, just a regular transfer.
-        (true, AccountType::EthImplicitAccount) => transfer_fee,
+        // Extra fee for the CreateAccount.
+        (true, AccountType::EthImplicitAccount) => {
+            #[cfg(feature = "protocol_feature_eth_implicit")]
+            {
+                transfer_fee + cfg.fee(ActionCosts::create_account).send_fee(sender_is_receiver)
```

Comment:
I believe this fee should only be applied on the first transfer from the account.

Reply:
This is applied on a transfer to the account, if it does not exist. I did analogous to the `NearImplicitAccount` below, which always include fees for `create_account` and `add_full_access_key`.

Reply:
I've just found a similar concern raised in: https://github.com/near/nearcore/pull/8568/files/e4cca01d862fd3634214fff20f148c07018e4e7a#r1109507652

Reply:
Ah sorry, I mixed up "create account" with "add access key". You're right.